src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -13,43 +13,85 @@ $breadcrumbPolygon: polygon(
   0% 100%
 );
 
-.osdHeaderBreadcrumbs {
-  /*
-  filter defines a custom filter effect by grouping atomic filter primitives!
-  here we are using Gaussian filter with stdDeviation for applying
-  border-radius on clipped background.
-  */
-
-  filter: url("../../public/assets/round_filter.svg#round");
+/* remove background color on autofocus for euiBreadcrumbs in Popover */
+.euiBreadcrumbs__inPopover {
+  .euiLink.euiLink--text:focus {
+    background: none;
+  }
+}
 
-  .osdBreadcrumbs {
+.osdHeaderBreadcrumbs {
+  /* common for both light & dark theme */
+  .osdBreadcrumbs,
+  &--dark .osdBreadcrumbs {
     clip-path: $breadcrumbPolygon;
-    background-color: $osdHeaderBreadcrumbGrayBackground;
     padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
 
     &:first-child {
       clip-path: $firstBreadcrumbPolygon;
     }
+  }
+
+  .osdBreadcrumbs {
+    background-color: $osdHeaderBreadcrumbGrayBackground;
 
     &:last-child {
       background-color: $osdHeaderBreadcrumbBlueBackground;
     }
   }
 
-  .euiBreadcrumbSeparator {
-    display: none;
-  }
-
+  /* only light mode */
```

Comment:
There's a little confusion with the naming - we have named these as if there's a default class, and then a more specific dark class that has some overrides (which implies that in dark mode the element would have `class="osdHeaderBreadcrumbs  osdHeaderBreadcrumbs--dark"`). But, in reality, we're only applying one or the other. If the class for light mode were named `osdHeaderBreadcrumbs--light`, you wouldn't need comments like this.

Reply:
comment is just for our understanding ! I thought one will be default and another would be dark